### PR TITLE
Af3 bug fixes

### PR DESCRIPTION
**Pull request by esilgard**
- commit 1 - update patient mapping table reference
- commit 2 - add try catch around sql server connections

### CODE REVIEW

**Review by ritchelong — APPROVED** [review]
LGTM!

File: active/af3-resynthesize-phi.py [hunk 1]
```diff
@@ -111,6 +112,7 @@ def populate_blobid_in_job_table(**kwargs):
                       "VALUES (%s, %s, %s, %s, %s) ".format(table=common_variables.AF3_RUNS_DETAILS))
 
     for creation_date in datecreated:
+      try:
```
Comment (WhiteAu) [inline active/af3-resynthesize-phi.py]: Would you move this to line 123? Generally I'd like to see us make our 'try' blocks as narrow in scope as possible.

File: active/af3-resynthesize-phi.py [hunk 2]
```diff
@@ -159,11 +166,16 @@ def _get_annotations_by_id_and_created_date(blobid, date):
...
+      return common_hooks.ANNOTATIONS_DB.get_records(src_select_stmt, parameters=(
                                              date, blobid, common_variables.REVIEW_BYPASSED_ANNOTATION_TYPE,
                                              common_variables.BRAT_REVIEWED_ANNOTATION_TYPE))
-
+    except OperationalError as e:
+        message = ("An OperationalError occured while trying to fetch annotations from the source data server"
+                   " for blobid {}".format(blobid))
+        print(message)
+        common_functions.log_error_and_failure_for_resynth_note_job(run_id, blobid, hdcpupdatedate, message,
+                                                                  "Get individual note annotations")
```
Comment (WhiteAu) [inline active/af3-resynthesize-phi.py:172]: Do we need to return something (eg: None) at this point? or raise the operational error and let the caller figure it out?

File: active/af3-resynthesize-phi.py [hunk 3]
```diff
@@ -225,14 +237,18 @@ def _get_patient_data_from_temp(blobid, hdcpupdatedate, patientid):
...
-            "{blobid}, hdcpupdatedate: {date}, patientId: {patientid}".format(
-                blobid=blobid, date=hdcpupdatedate, patientid=patientid))
-    return ret
+      if ret[0] is None:
+          print(
+              "No Real Patient Name Data found in Temp DB for blobID: "
+              "{blobid}, hdcpupdatedate: {date}, patientId: {patientid}".format(
+                  blobid=blobid, date=hdcpupdatedate, patientid=patientid))
+      return ret
+    except OperationalError as e:
```
Comment (WhiteAu) [inline active/af3-resynthesize-phi.py:238]: move the except statement up to above line 243 please

**Review by WhiteAu — APPROVED** [review]
looks good. some general suggestions on try/except below

File: active/af3-resynthesize-phi.py (hunk 1 above)
Comment (esilgard) [reply active/af3-resynthesize-phi.py]: absolutely - thanks for seeing that.

Comment (esilgard) [reply active/af3-resynthesize-phi.py]: *edit*
this is where the call to SQL Server is - line 123 is the call to the Postgres database, which we haven't seemed to have issues with

File: active/af3-resynthesize-phi.py (hunk 2 above)
Comment (esilgard) [reply active/af3-resynthesize-phi.py:172]: yeah, I'll add a return None

File: active/af3-resynthesize-phi.py (hunk 3 above)
Comment (esilgard) [reply active/af3-resynthesize-phi.py:238]: roger.

File: active/af3-resynthesize-phi.py (hunk 1 above)
Comment (esilgard) [reply active/af3-resynthesize-phi.py]: is there another place you see to put the try catch for the sql call?

File: active/af3-resynthesize-phi.py (hunk 2 above)
Comment (ddong63) [reply active/af3-resynthesize-phi.py:172]: Do we want to retry if there is an OperationalError? A lot of the times it runs thru when you rerun the dag.

**Review by ddong63 — APPROVED** [review]

**Comment by ddong63** [comment]
Conflicts needs to be resolved before merging

File: active/af3-resynthesize-phi.py (hunk 2 above)
Comment (esilgard) [reply active/af3-resynthesize-phi.py:172]: but if it doesn't pass on another attempt than we're back where we started and not moving onto the next task

**Pull request merged**